packages/core/contracts/oracle/implementation/VoteTiming.sol

```diff
@@ -32,7 +32,7 @@ library VoteTiming {
      * @return roundId defined as a function of the currentTime and `phaseLength` from `data`.
      */
     function computeCurrentRoundId(Data storage data, uint256 currentTime) internal view returns (uint256) {
-        uint256 roundLength = data.phaseLength.mul(uint256(VotingInterface.Phase.NUM_PHASES_PLACEHOLDER));
```

Comment:
Unfortunately, some state had to move to the VotingAncillaryInterface to get the inheritance tree to compile correctly with truffle.